ios/ReactTestApp/AppDelegate.swift

```diff
@@ -4,10 +4,8 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
     private weak var application: UIApplication?
 
-    // TODO: `window` no longer exists but RedBox has not been updated to
-    //       support UIScene yet. Revisit when we bump React Native.
     @objc var window: UIWindow? {
-        get { application?.windows.first }
+        get { RCTKeyWindow() }
```

Comment:
This change may only be valid for React Native 0.73+

Reply:
`RCTKeyWindow` changes between versions. Would it make sense to copy its latest iteration here?

Reply:
You mean copy the implementation?

Reply:
Yes

Reply:
Thanks for the suggestion while I was out. Looks like it worked :)